src/graph_serializer.cc

```diff
@@ -4,12 +4,19 @@
 
 #include "command.h"
 #include "command_graph.h"
+#include "task_manager.h"
 
 namespace celerity {
 namespace detail {
 
 	void graph_serializer::flush(task_id tid) { flush(cdag.task_commands(tid)); }
 
+	bool flushed_manually(const abstract_command* const cmd) {
```

Comment:
Is the semantic overhead of dealing with this worth it compared to simply flushing it?

Reply:
Interesting, I don't remember this being the reason for not flushing the init commands (it might've been, although I thought it was more of a premature optimization). Possible workaround: Retroactively invoke the callback for all existing tasks upon registering.

Regarding the current workaround:
- Why is it called `flushed_manually` when initial epochs aren't flushed at all?
- Can't we just rely on `cmd->is_flushed()` to handle this instead?

Reply:
This is now renamed. In a live discussion with @psalz we decided that we want to flush the init task like any other eventually, but the refactoring would make this PR unnecessarily complicated.